### PR TITLE
Added new sample and its template.json

### PR DESCRIPTION
**Pull request by Antibodyy**
Very interesting and promising project. 
The link to the image https://image.shutterstock.com/image-vector/omr-sheet-icon-clipart-image-260nw-2029607780.jpg

### CODE REVIEW

**Comment by Antibodyy** [comment]
Do add the hacktoberfest label, since I forgot to do so :P

**Comment by Antibodyy** [comment]
How about now @Udayraj123 ? I added a different sample and have calibrated its template as well.

**Comment by Udayraj123** [comment]
Can you share the output of these
```
 python3 main.py -i samples/sample2
```
```
 python3 main.py -i samples/sample7
```

**Comment by Udayraj123** [comment]
Can you add reference url to source images?

**Comment by Udayraj123** [comment]
Rename 
```
samples/sample2/omr_test_01.png --> samples/sample2/AdrianSample/adrian_omr_2.png
samples/sample7/anti_samples/omr-sheet-1000x1000.jpg --> samples/sample7/simple_omr_sheet.jpg
```

**Comment by Antibodyy** [comment]
todel1
todel2
todel3

File: src/defaults/config.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
   },
   "outputs": {
     "show_image_level": 0,
-    "save_image_level": 2,
+    "save_image_level": 4,
```
Comment (Udayraj123) [inline src/defaults/config.json:25]: remove this change. rest good to go

**Review by Udayraj123 — APPROVED** [review]

**Pull request merged**